README.md

```diff
@@ -3,3 +3,5 @@
 Utilities and tools for AMD SEV memory encryption technologies:
 
  - [snp.sh](docs/snp.md) - Setup a host system, launch and test a SNP guest
+
+ - [Launch_SVSM.sh](docs/svsm.md) - This utility is designed to set up and configure a system to utilize Coconut-SVSM (Secure Virtual Secure Manager) in conjunction with AMD-SEV (Secure Encrypted Virtualization) technologies. It automates the installation, configuration, and validation processes needed to ensure that the system is ready for secure virtual machine operations using SVSM and AMD-SEV
```

Comment:
It took me a second to understand what the point of each script was. User's are intended to only use Launch_SVSM.sh and SVSM_setup.sh is more of a helper script you have to help install dependencies. 

I'd rename the scripts:
Launch_SVSM.sh -> svsm.sh
SVSM_setup.sh -> svsm_helper.sh or something like that

It just clarifies the point of each script. the way it's set up right now it seems Launch is used for launching and setup is using for setting up.

Reply:
Good suggestion. Agreed your suggestions. I named the file based on its contents, but simpler is better. The SVSM_setup.sh script can also run individual images for debugging. However, we don't use those steps in sev_utils. I'll update the coconut-svsm documentation to highlight this feature, as many new members are facing compilation issues.